apps/judicial-system/web/src/components/DefenderInfo/DefenderInfo.tsx

```diff
@@ -219,6 +245,15 @@ const DefenderInfo: React.FC<Props> = (props) => {
           {renderTooltip()}
         </Text>
       </Box>
+      {defenderNotFound && (
+        <Box marginBottom={3} data-testid="defenderNotFound">
+          <AlertMessage
+            type="warning"
+            title={formatMessage(getTranslations().defenderNotFoundTitle)}
```

Comment:
Consider defining a single pair of defender-not-found strings in the the core namespace, as it is very unlikely that the strings will differ based on role and/or case type.